pkg/otlp/config_test.go

```diff
@@ -16,8 +16,11 @@ import (
 
```

Comment:
`"fmt"` needs to be added to the import list here

Reply:
Done